src/script/builder/tir/var.cc

```diff
@@ -63,6 +64,11 @@ TVM_STATIC_IR_FUNCTOR(Namer, vtable)
       Namer::Name(var->var, name);
     });
 
+TVM_REGISTER_GLOBAL("script.builder.tir.Buffer")
+  .set_body_typed([](Array<PrimExpr> shape, DataType dtype, String name, String storage_scope){
+    return Buffer_(shape, dtype, name, storage_scope);
+  });
```

Comment:
i suppose it could be simplified as:

```suggestion
TVM_REGISTER_GLOBAL("script.builder.tir.Buffer")
  .set_body_typed(Buffer_);
```